### PR TITLE
fix(platform): Fix regression from #15986

### PR DESCRIPTION
**Pull request by leedongwei**
Refs ISSUE-644

### CODE REVIEW

**Comment by leedongwei** [comment]
2nd commit for CSS is hard to test on `dev`. I need to push to `staging` to take a look first before we can merge it in.

File: src/sentry/static/sentry/less/group-detail.less [hunk 1]
```diff
@@ -1735,7 +1735,8 @@ span.val {
     padding: 0 0 0 15px;
 
     .val-array-item {
-      display: block;
+      display: flex;
+      flex-direction: row;
```
Comment (BYK) [inline src/sentry/static/sentry/less/group-detail.less]: Why this change?

**Review by BYK — APPROVED** [review]
Thanks for fixing my mess!

File: src/sentry/static/sentry/less/group-detail.less (hunk 1 above)
Comment (leedongwei) [reply src/sentry/static/sentry/less/group-detail.less]: This might have to do with some CSS changes from Dora

If it's `display: block`:
image

If it's `display: flex`:
image

I am testing this change on `staging` before merging.

Comment (leedongwei) [reply src/sentry/static/sentry/less/group-detail.less]: Looks good on `staging`.

Comment (BYK) [reply src/sentry/static/sentry/less/group-detail.less]: That's kind of weird as I haven't touched on any CSS. Anyway, if it looks good I'm good.
